<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOption</class>
 <widget class="QDialog" name="DialogOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_working_directory">
     <property name="text">
      <string>Working directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="directory_layout">
     <item>
      <widget class="QLineEdit" name="working_directory">
       <property name="toolTip">
        <string>Working directory for exchanging files with the device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="select_directory">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Working directory for exchanging files with the device</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="title">
      <string>Console</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_console_font">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="4">
       <widget class="QLabel" name="label_output">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;def function(self):&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;for j in range(10):&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;print(j)&lt;br&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFontComboBox" name="combo_font">
        <property name="toolTip">
         <string>Select font family for the output console</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="fontFilters">
         <set>QFontComboBox::MonospacedFonts</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spin_font_size">
        <property name="toolTip">
         <string>Select the font size for the output console</string>
        </property>
        <property name="minimum">
         <number>7</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button_backcolor">
        <property name="toolTip">
         <string>Select the default background color for the output console</string>
        </property>
        <property name="text">
         <string>Background color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>148</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button_forecolor">
        <property name="toolTip">
         <string>Select the default text color for the output console</string>
        </property>
        <property name="text">
         <string>Text color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogOption</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogOption</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
